src/content/docs/synthetics/synthetic-monitoring/pages/synthetic-monitoring-aggregate-monitor-metrics.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Synthetic monitoring: Aggregate monitor metrics'
+title: 'Synthetic monitoring: SLA report'
```

Comment:
Maybe we should explode this acronym @austin-schaefer? I only say that because someone asked what SLA was in our meeting today. :)

Reply:
I prefer the acronym I think, but a fair callout!